webdriver/tests/bidi/script/call_function/arguments.py

```diff
@@ -112,15 +112,20 @@ async def test_default_arguments(bidi_session, top_context):
             "(func) => func()",
             {"type": "boolean", "value": False},
         ),
+        (
+            "document.createElement('div')",
+            "(node) => node.tagName",
```

Comment:
Is it any different from just a string serialization?

Reply:
this is not the test you want to look for. I've only moved it up so it stays in the same order as listed for the serialization in the BiDi spec. The failure with Chrome is in the other test as added below.